spec/beaker/host_spec.rb

```diff
@@ -553,6 +553,32 @@ module Beaker
       end
     end
 
+    context 'do_rsync_to' do
+      it 'do_rsync_to logs info and call Rsync class' do
+        create_files(['source'])
+        logger = host[:logger]
+        @options = { :logger => logger }
+        args = [ 'source', 'target', {:ignore => ['.bundle']} ]
+
+        key = host['ssh']['keys']
+        if key.is_a? Array
+          key = key.first
+        end
+
+        rsync_args = [ 'source', 'target', ['-az', "-e \"ssh -i #{key} -p 22 -o 'StrictHostKeyChecking no'\"", "--exclude '.bundle'"] ]
+
+        expect( host ).to receive(:to_s).and_return('host.example.org')
+        #expect(Rsync.host).to eq('root@host.example.org')
```

Comment:
I'm having an issue to test that Rsync.host gets set to this hostname. During the test run, it comes back as nil, presumably due to an execution order issue.

Does anyone have any ideas?